README.md

```diff
@@ -25,43 +25,43 @@ From your shell or command line:
 
-If you already have existing policies in your B2C tenant, feel free to re-use those.  No need to create new ones just for this sample.
+If you already have existing policies in your B2C tenant, feel free to re-use those. No need to create new ones just for this sample.
 
 ### Step 5: Create your own application
 
-Now you need to create your own appliation in your B2C tenant, so that your app has its own client ID.  You can do so following [the generic instructions here](https://azure.microsoft.com/documentation/articles/active-directory-b2c-app-registration).  Be sure to include the following information in your app registration:
+Now you need to create your own appliation in your B2C tenant, so that your app has its own client ID.  You can do so following [the generic instructions here](https://azure.microsoft.com/documentation/articles/active-directory-b2c-app-registration). Be sure to include the following information in your app registration:
 
 - Enable the **Web App/Web API** setting for your application.
-- Add **two** redirect_uris for your app, one for each policy used in the sample.  Their values should take the form `https://localhost:44316/{your-policy-name}`, where your policy names begin with `b2c_1_`.
+- Add the redirect URI for your app: `https://localhost:44316/signin-oidc`.
 - Copy the client ID generated for your application, so you can use it in the next step.
 
 ### Step 6: Configure the sample to use your B2C tenant
 
-Now you can replace the app's default configuration with your own.  Open the `config.json` file and replace the following values with the ones you created in the previous steps.  You can leave the `AadInstance` and `PostLogoutRedirectUri` values unchanged.
+Now you can replace the app's default configuration with your own.  Open the `appsettings.json` file and replace the following values with the ones you created in the previous steps.  You can leave the `AAdB2CInstance` value unchanged.
 
 ```json
 {
-    "AzureAd": {
-        "ClientId": "90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6",
-        "Tenant": "fabrikamb2c.onmicrosoft.com",
-        "AadInstance": "https://login.microsoftonline.com/{0}/v2.0/.well-known/openid-configuration?p={1}", // This is the public instance of Azure AD
-        "PostLogoutRedirectUri": "https://localhost:44316/",
-
-        "SignUpPolicyId": "B2C_1_sign_up",
-        "SignInPolicyId": "B2C_1_sign_in",
+  "Authentication": {
+    "AzureAdB2C": {
+      "ClientId": "90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6",
+      "Tenant": "fabrikamb2c.onmicrosoft.com",
+      "AAdB2CInstance": "https://login.microsoftonline.com/tfp/fabrikamb2c.onmicrosoft.com/b2c_1_susi/v2.0/.well-known/openid-configuration", // This is the public instance of Azure AD
```

Comment:
We should not hardcode this but rather build it dynamically based on the tenant, policy name and AzureAdB2COptions.Authority